python/tests/sbp/client/test_driver.py

```diff
@@ -38,41 +42,44 @@ def tcp_server(handler):
   return (ip, port)
 
 def test_tcp_logger():
-  handler = tcp_handler(MsgPrintDep(text='abc').to_binary())
+  handler = tcp_handler(MsgPrintDep(text=b'abc').to_binary())
   ip, port = tcp_server(handler)
   port = "socket://%s:%s" % (ip, port)
   baud = 115200
   t0 = time.time()
   sleep = 0.1
-  def assert_logger(s):
+  temp_d = {'assert_logger_called': False}  # nonlocal doesn't work in python 2.7
+  def assert_logger(s, **metadata):
+    temp_d['assert_logger_called'] = True
     assert s.preamble==0x55
     assert s.msg_type==0x10
     assert s.sender==66
     assert s.length==3
-    assert s.payload=='abc'
+    assert s.payload==b'abc'
     assert s.crc==0xDAEE
   with PySerialDriver(port, baud) as driver:
     with Handler(Framer(driver.read, None, verbose=False)) as link:
       link.add_callback(assert_logger)
       while True:
-        if time.time() - t0 < sleep:
+        if time.time() - t0 > sleep:
           break
+  assert temp_d['assert_logger_called']
 
 BASE_STATION_URI = "http://broker.testing.skylark.swiftnav.com"
 
 @activate
 def test_http_test_pass():
   assert is_enabled()
-  msg = MsgPrintDep(text='abcd')
+  msg = MsgPrintDep(text=b'abcd')
```

Comment:
Should these be updated to use some non-deprecated message?

Reply:
In principle a valid idea but since MsgPrintDep is used extensively in the tests and it is not directly related to Python 3, I think it's better done in separate task.

Reply:
As a separate task, this HTTP driver test likely needs to be scrapped completely.  

This is left-over from our initial concept of skylark over a proprietary HTTP stream initiated by the console.

Reply:
ok, I see that it too has the same problems that I fixed in `test_tcp_logger`, i.e., it doesn't actually test that the callback was executed and the time condition is broken (inverted). But if it's going to be scrapped, I won't fix it in this context.

Reply:
@denniszollo so the obsolete part is `test_http_test_pass`, `test_http_test_fail`, `test_http_test_pass_streaming`, `test_http_test_pass_retry` (and some related imports and definitions), correct?

Reply:
I think @denniszollo is saying we can remove `HTTPDriver` and all the related tests.

Reply:
ok, in a separate task/pr